packages/apps/stock/src/components/templates/supplierArrival/SupplierArrivalLineButtons/SupplierArrivalLineButtons.js

```diff
@@ -53,7 +53,7 @@ const SupplierArrivalLineButtons = ({
         stockMoveLineId: supplierArrivalLine.id,
         version: supplierArrivalLine.version,
         realQty: realQty,
-        conformity: conformity.id,
```

Comment:
API request is only asking for value of conformity, conformity object here is composed of id and name to match with picker. Please leave id here